electrum/gui/qt/wizard/server_connect.py

```diff
@@ -110,3 +110,4 @@ def __init__(self, parent, wizard):
     def apply(self):
         self.wizard_data['autoconnect'] = self.sw.server_e.text().strip() == ''
         self.wizard_data['server'] = self.sw.server_e.text()
+        self.wizard_data['one_server'] = False
```

Comment:
We can avoid duplicating the default value of the config setting, by delegating it to the config:
```suggestion
        self.wizard_data['one_server'] = self.wizard.config.NETWORK_ONESERVER
```